src/promptflow-devkit/promptflow/orchestrator/_chat_group_orchestrator.py

```diff
@@ -0,0 +1,222 @@
+from typing import Optional, List, Mapping, Dict, Any
+from promptflow._sdk.entities._chat_group._chat_role import ChatRole
+from promptflow._proxy._base_executor_proxy import AbstractExecutorProxy
+from promptflow.executor._result import LineResult
+from promptflow.storage import AbstractRunStorage
+from promptflow.batch._batch_inputs_processor import BatchInputsProcessor
+from promptflow._utils.execution_utils import apply_default_value_for_input
+from promptflow._proxy._proxy_factory import ProxyFactory
+from promptflow._utils.logger_utils import bulk_logger
+from promptflow._constants import CONVERSATION_HISTORY_EXPRESSION, CONVERSATION_HISTORY_OUTPUT_KEY
+from promptflow.orchestrator._errors import (
+    InvalidChatRoleCount,
+    MissingConversationHistoryExpression,
+    MultipleConversationHistoryInputsMapping
+)
+
+
+class ChatGroupOrchestrator:
```

Comment:
NIT: maybe rename directory `orchestrator` to `_orchestrator` if you don't want to make it public. Anyway, there is no import in current `__init__.py`

Reply:
+ 1

Reply:
Change it to private.